### PR TITLE
[docs] 2.1 DB1

### PR DESCRIPTION
**Pull request by jamiltz**

### CODE REVIEW

**Comment by jamiltz** [comment]
@hideki here some stubs for the 2.1 docs, could you add the java examples?

iOS PR: https://github.com/couchbase/couchbase-lite-ios/pull/2154

File: docs/modules/ROOT/pages/java.adoc [hunk 1]
```diff
@@ -857,13 +830,16 @@ explicitly enable them in the load balancer's configuration (see
 https://developer.couchbase.com/documentation/mobile/1.5/guides/sync-gateway/nginx/index.html[Load
 Balancers]).
```
Comment (rizwanbpatel) [inline docs/modules/ROOT/pages/java.adoc]: This needs to be updated to or this is as expected?
https://developer.couchbase.com/documentation/mobile/2.0/installation/sync-gateway/index.html

File: docs/modules/ROOT/pages/java.adoc [hunk 2]
```diff
@@ -827,10 +801,9 @@ Gateway
 https://developer.couchbase.com/documentation/mobile/1.5/installation/sync-gateway/index.html[installation
```
Comment (rizwanbpatel) [inline docs/modules/ROOT/pages/java.adoc]: Correct link for 2.0 Sync Gateway should be?
https://developer.couchbase.com/documentation/mobile/2.0/installation/sync-gateway/index.html [installation guide]

File: docs/modules/ROOT/pages/java.adoc [hunk 3]
```diff
@@ -857,13 +830,16 @@ explicitly enable them in the load balancer's configuration (see
 https://developer.couchbase.com/documentation/mobile/1.5/guides/sync-gateway/nginx/index.html[Load
```
Comment (rizwanbpatel) [inline docs/modules/ROOT/pages/java.adoc]: Current documentation link for the Load Balancer should be as below.
https://developer.couchbase.com/documentation/mobile/current/guides/sync-gateway/nginx/index.html

File: docs/modules/ROOT/pages/_partials/pre-release-instructions.adoc [hunk 4]
```diff
@@ -0,0 +1,26 @@
...
+    }
+}
+----
+* Next, add the following in the `dependencies` section of the
+application's *build.gradle* (the one in the *app* folder).
++
+[source,groovy]
+----
+dependencies {
+    compile 'com.couchbase.lite:couchbase-lite-android:2.1DB001'
```
[the remark this answers is not in the document]
Comment (pasin) [reply docs/modules/ROOT/pages/_partials/pre-release-instructions.adoc]: I will need to verify this and I will let you know.

**Pull request closed without merging**